### ISSUE TITLE
Research high a11y scan failure rate

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#672]
In a [recent primary snapshot](https://docs.google.com/spreadsheets/d/1TnhgpnpSkyYXZrwmD7jdVKqKXG7wS9HkCtj8gVB18UY/edit#gid=818633306), 1/3 of websites, which all had successful primary scans, had failed accessibility scans.  We should try to understand why it's so high and redress it.

**Comment by gbinal** [comment]
There's good progress on this.  We are getting a fresh export to then analyze again.

**Comment by gbinal** [comment]
The recent changes are better, but there's still a high error rate.  [I'm seeing](https://docs.google.com/spreadsheets/d/13SD6FwhoJ29vJvG9AnLSr9FJIrZv_6055O5GvekvIUI/edit#gid=525942424) 2,024 of 15k getting errors and not generating data.  1956 are unknown errors, 68 are timeout errors.

**Comment by gbinal** [comment]
There's been a lot more progress and we're now down to 400 or so urls not recording data.  That's worth future investigating but is enough progress that we can close this for now.
